### ISSUE TITLE
AntiVirus 1.0 - PHP Backdoor Detection Bypass

### ISSUE THREAD

**Issue by anantshri** [wpscanteam/wpscan#824]
One more bad apple : https://wpvulndb.com/vulnerabilities/6854 it claims the issue is still not fixed. as no fix version specified.

The article references ultimately points to this https://github.com/MustLive/BWA/blob/master/bwa.php as not being detected.

where as https://plugins.svn.wordpress.org/antivirus/tags/1.3.8/antivirus.php has following code

```
private static function _php_match_pattern()
	{
		return '/(assert|file_get_contents|curl_exec|popen|proc_open|unserialize|eval|base64_encode|base64_decode|create_function|exec|shell_exec|system|passthru|ob_get_contents|file|curl_init|readfile|fopen|fsockopen|pfsockopen|fclose|fread|file_put_contents)\s*?\(/';
	}
```

Seems like this should be captured.

There is no verified status also can you check this one also and validate if we have this or not.

**Comment by ethicalhack3r** [comment]
Detected bwa.php for me in version 1.1

![screenshot_1018](https://cloud.githubusercontent.com/assets/323113/7584213/1aaa3e18-f89c-11e4-9083-c69fc4d869b9.png)

Could not test 1.0 as download no longer exists - https://downloads.wordpress.org/plugin/antivirus.1.0.zip (wasn't tagged in svn)

Looking at the diff between 1.0 and 1.1 shows no changes which improve detection, making me assume that this is a false positive - https://plugins.trac.wordpress.org/changeset?sfp_email=&sfph_mail=&reponame=&new=425039%40antivirus&old=323875%40antivirus&sfp_email=&sfph_mail=

**Comment by anantshri** [comment]
can we atleast mark this as fixed in 1.1

**Comment by ethicalhack3r** [comment]
done :)
